build.py

```diff
@@ -330,7 +337,26 @@ def build(version=None,
             build_command += "-X main.branch={} ".format(branch)
             build_command += "-X main.commit={}\" ".format(get_current_commit())
         build_command += c
-        out = run(build_command, shell=True)
+        
+        try_bootstrap = False
```

Comment:
Bootstrapping shouldn't be necessary, I would prefer it to just error instead of attempt to make a change to the system. We can do the bootstrapping in docker and then this script will just fail if its not setup correctly. Then we will just need to go fix the Dockerfile.